raisinbread/RB_files/RB_parameter_type.sql

```diff
@@ -237,7 +237,6 @@ INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`,
 INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`, `RangeMin`, `RangeMax`, `SourceField`, `SourceFrom`, `SourceCondition`, `Queryable`, `IsFile`) VALUES (235,'series_description','text','series_description magically created by NeuroDB::File',NULL,NULL,NULL,'parameter_file',NULL,0,0);
 INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`, `RangeMin`, `RangeMax`, `SourceField`, `SourceFrom`, `SourceCondition`, `Queryable`, `IsFile`) VALUES (236,'magnetic_field_strength','text','magnetic_field_strength magically created by NeuroDB::File',NULL,NULL,NULL,'parameter_file',NULL,0,0);
 INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`, `RangeMin`, `RangeMax`, `SourceField`, `SourceFrom`, `SourceCondition`, `Queryable`, `IsFile`) VALUES (237,'dicom_0x0051:el_0x1016','text','dicom_0x0051:el_0x1016 magically created by NeuroDB::File',NULL,NULL,NULL,'parameter_file',NULL,0,0);
-INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`, `RangeMin`, `RangeMax`, `SourceField`, `SourceFrom`, `SourceCondition`, `Queryable`, `IsFile`) VALUES (238,'header','text','header magically created by NeuroDB::File',NULL,NULL,NULL,'parameter_file',NULL,0,0);
```

Comment:
Is this line supposed to be removed? The description just says it's upgrading the migration file.

Reply:
yes, it was a missed patch that I run